keps/sig-api-machinery/2161-apiserver-default-labels/kep.yaml

```diff
@@ -0,0 +1,31 @@
+title: Immutable label selectors for all namespaces 
+kep-number: 2161
+authors:
+  - "@thockin"
+  - "@jayunit100"
+  - "@rikatz"
+  - "@abhiraut"
+  - "@andrewsykim"
+owning-sig: sig-api-machinery
+participating-sigs:
+  - sig-network
+status: provisional
+creation-date: 2020-11-22
+reviewers:
+  - "@liggitt"
+  - "@nikhita"
+  - "@thockin"
+approvers:
+  - "@liggitt"
```

Comment:
someone from api-machinery as an approver would make sense as well... @deads2k is already reviewing, might make sense